modules/apps/headless/headless-admin-user/headless-admin-user-test/src/testIntegration/java/com/liferay/headless/admin/user/resource/v1_0/test/BaseUserAccountResourceTestCase.java

```diff
@@ -984,29 +984,7 @@ public void testDeleteAccountUserAccountsByExternalReferenceCodeByEmailAddress()
 	public void testPostAccountUserAccountsByExternalReferenceCodeByEmailAddress()
 		throws Exception {
 
-		@SuppressWarnings("PMD.UnusedLocalVariable")
-		UserAccount userAccount =
-			testPostAccountUserAccountsByExternalReferenceCodeByEmailAddress_addUserAccount();
-
-		assertHttpResponseStatusCode(
-			204,
-			userAccountResource.
-				postAccountUserAccountsByExternalReferenceCodeByEmailAddressHttpResponse(
-					userAccount.getExternalReferenceCode(), null));
-
-		assertHttpResponseStatusCode(
-			404,
-			userAccountResource.
-				postAccountUserAccountsByExternalReferenceCodeByEmailAddressHttpResponse(
-					userAccount.getExternalReferenceCode(), null));
-	}
-
-	protected UserAccount
-			testPostAccountUserAccountsByExternalReferenceCodeByEmailAddress_addUserAccount()
-		throws Exception {
-
-		throw new UnsupportedOperationException(
-			"This method needs to be implemented");
+		Assert.assertTrue(false);
```

Comment:
can you check why now this test is empty? Because in this way we are losing this test (and if it's not overridden it will always fail)